packages/shared/src/dto/lap.ts

```
@@ -0,0 +1,13 @@
+
+export interface LapDataResponseDTO {
+  data: ILapData[];
+  message: string;
+  uptime: string;
+}
+
+export interface LapHealthResponseDTO {
+  date: Date;
+  message: string;
+  uptime: string;
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)